ntoskrnl/ke/i386/kiinit.c

```diff
@@ -536,7 +536,8 @@ KiInitializeKernel(IN PKPROCESS InitProcess,
     else
     {
         /* FIXME */
-        DPRINT1("SMP Boot support not yet present\n");
+        DPRINT1("Starting CPU#%u - you are brave!\n", Number);
```

Comment:
```suggestion
        DPRINT1("Starting CPU#%u - you are brave\n", Number);
```

Reply:
#BringPersonalityBackToTheReactOSSourceTree

Reply:
If you think the personallity is needed, then at least strip the exclamation mark. I will accept that as a solution as well.